### DISCUSSION TITLE
Start Printer

### DISCUSSION THREAD

**Discussion by DanielOveridick** [aria1th/litematica-printer#48] [General]
Hey,

how do I start the printer?
I'm playing the 1.19 version of GrieferGames.

I use toggle easyPrint but nothing happens.
Printer is enabled.

Thanks in advance for helping.

**Comment by aria1th** [comment]
I'm not sure what is `easyPrint`, maybe `easyPlace`? you need to use `easyPlaceUseKey` such as 'right click'
